src/components/Title.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Title = styled.div`
+  display: flex;
+  justify-content: center;
+  font-size: ${({ fontSize }) => fontSize}px;
+`;
+export default ({ title, fontSize }) => {
+  return <Title fontSize={fontSize}>{title}</Title>;
```

Comment:
속성과 그 안에 들어가는 변수명이 같다면 축약할 수 있다고 합니다..! 

```suggestion
  return <Title {...{fontSize}}>{title}</Title>;
```